root/usr/bin/nsdc-cmd

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2018 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+# Execute a command inside the containter
+
+. /usr/share/nethserver-dc/nsdc-functions
```

Comment:
Why not to merge the function definitions here? Do plan to use them directly from other future scripts?

Reply:
Yes, I started creating the functions as a library to be reusable.
But I think we can merge into the command.